Project/Assets/Editor/Twine Importer/TwineImporterUI.cs

```diff
@@ -18,21 +19,9 @@ public class TwineImporterUI : EditorWindow
 	[MenuItem("Assets/Import Twine Data...")]
 	static void ShowTwineImportWindow ()
 	{
-
-		// if triggered while a text asset is selected, populate it as the target file
-		TextAsset selectedFile = null;
-		if (Selection.activeObject != null)
-		{
-			var filePath = AssetDatabase.GetAssetPath (Selection.activeObject);
-			if (Path.GetExtension (filePath).Contains ("json"))
-			{
-				selectedFile = (TextAsset) Selection.activeObject;
-			}
-		}
-
 		// create and show window
 		var window = EditorWindow.GetWindow<TwineImporterUI> ();
-		window.targetFile = selectedFile;
+		window.Show ();
```

Comment:
This is more explicit (and thus probably clearer), but just for the record, `EditorWindow.GetWindow` automatically shows the window if it isn't in front.